packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -249,6 +249,48 @@ describe('<Autocomplete />', () => {
       expect(getByRole('listbox')).to.have.property('scrollTop', 0);
     });
 
+    // https://github.com/mui/material-ui/issues/36212
+    it('should preserve scrollTop position of the listbox when adding new options on mobile', function test() {
+      if (/jsdom/.test(window.navigator.userAgent)) {
+        this.skip();
+      }
+      function getOptions(count) {
+        return Array(count)
+          .fill('item')
+          .map((value, i) => value + i);
+      }
+
+      const { getByRole, getAllByRole, setProps } = render(
+        <Autocomplete
+          open
+          options={getOptions(5)}
+          renderInput={(params) => <TextField {...params} />}
+          ListboxProps={{ style: { padding: 0, maxHeight: '100px' } }}
+          PopperComponent={(props) => {
```

Comment:
Why is a custom Popper component added? This does not seem necessary.

Reply:
You right thanks! (copy/paste 😆 )